.github/workflows/ci.yml

```diff
@@ -332,11 +332,10 @@ jobs:
     name: Smoke Test - Android ${{ matrix.api-level }} Unity ${{ matrix.unity-version }}
     runs-on: macos-latest
     strategy:
-      max-parallel: 5
+      max-parallel: 7
```

Comment:
Should we just drop this and let it use as many agents as there are available?

Reply:
ok